generator/src/test/java/com/scottlogic/deg/generator/cucumber/features/operators/string/ShorterThan.feature

```diff
@@ -251,3 +246,15 @@ Feature: User can specify that a string length is lower than, a specified number
       | foo  |
       | null |
       | "x"  |
+
+  Scenario: shorterThan with maximum permitted value should be successful
+    Given foo is shorter than 1000
+    And the generation strategy is random
+    And the generator can generate at most 1 rows
+    And foo is anything but null
+    Then foo contains strings of length between 1 and 1000 inclusively
+
+  @ignore #awaiting rest of #848
+  Scenario: shorterThan with value larger than maximum permitted should fail with an error message
+    Given foo is shorter than 1001
+    Then the profile is invalid because "'value' property is out of upper bound"
```

Comment:
Should 1001 be an allowable value for shorterThan? It would result in strings up to the max allowable length but not longer

Reply:
Yes for shorterThan, but not any other string operator.

shorterThan 1001 means length <= 1000, so it's fine to say that.